src/test/java/org/takes/rq/multipart/RqMtFakeTest.java

```diff
@@ -115,36 +108,35 @@ public void throwsExceptionOnInvalidContentTypeHeader() throws IOException {
 
     /**
      * RqMtFake can parse http body.
-     * @throws IOException If some problem inside
+     * @throws Exception
      */
     @Test
-    public void parsesHttpBody() throws IOException {
+    public void parsesHttpBody() throws Exception {
         final String body = "40 N Wolfe Rd, Sunnyvale, CA 94085";
         final String part = "t4";
         final RqMultipart multi = new RqMtFake(
             new RqFake(),
             new RqWithHeaders(
                 new RqFake("", "", body),
-                RqMtFakeTest.contentLengthHeader(
-                    (long) body.getBytes().length
-                ),
-                RqMtFakeTest.contentDispositionHeader(
-                    String.format("form-data; name=\"%s\"", part)
-                )
+                new FormattedText(
+                    "Content-Length: %s", body.getBytes().length
+                ).asString(),
+                new FormattedText(
+                    "Content-Disposition: form-data; name=\"%s\"", part
+                ).asString()
             ),
             new RqWithHeaders(
                 new RqFake("", "", ""),
-                RqMtFakeTest.contentLengthHeader(0L),
-                RqMtFakeTest.contentDispositionHeader(
-                    String.format(RqMtFakeTest.FORM_DATA, "a.rar")
-                )
+                "Content-Length: 0",
+                // @checkstyle LineLengthCheck (1 line)
+                "Content-Disposition: form-data; name=\"data\"; filename=\"a.rar\""
```

Comment:
@Serranya use `JoinedText` to break this line and remove the checkstyle supression.